broadcast.go

```diff
@@ -29,6 +29,11 @@ func (b *memberlistBroadcast) Invalidates(other Broadcast) bool {
 	return b.node == mb.node
 }
 
+// memberlist.NamedBroadcast optional interface
```

Comment:
There's a very small change that should be made to serf to have it skip the O(N) traversal codepath:
```
diff --git a/serf/broadcast.go b/serf/broadcast.go
index d20728f..751cf18 100644
--- a/serf/broadcast.go
+++ b/serf/broadcast.go
@@ -16,6 +16,9 @@ func (b *broadcast) Invalidates(other memberlist.Broadcast) bool {
        return false
 }
 
+// implements memberlist.UniqueBroadcast
+func (b *broadcast) UniqueBroadcast() {}
+
 func (b *broadcast) Message() []byte {
        return b.msg
 }
```